src/main/java/io/camunda/testing/utils/RecordStreamManager.java

```diff
@@ -0,0 +1,25 @@
+package io.camunda.testing.utils;
+
+import org.camunda.community.eze.RecordStreamSource;
+
+public abstract class RecordStreamManager {
```

Comment:
:wrench: Maybe rename to `RecordStreanStore` I am not sure about calling it manager.

Reply:
Also not sure whether the class is helpful, to be honest. Now BPMNAssert and this class are sort of tied together, but the coupling is less obvious. This is not to be understood as a veto though. If you have ideas on how this class will develop going forward, I am happy to discuss these. But right now it looks like you separated concerns, but not the underlying coupling.